modules/persistent-storage-csi-group-snapshots-restore.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-group-snapshots.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-group-snapshots-restore_{context}"]
+= Restoring a volume group snapshot
+
+You can use the `VolumeGroupSnapshot` custom resource definition (CRD) content to restore the existing volumes to a previous state.
+
+To restore existing volumes, you can request a new persistent volume claim (PVC) to be created from a `VolumeSnapshot` object that is part of a `VolumeGroupSnapshot`. This triggers provisioning of a new volume that is populated with data from the specified snapshot. Repeat this process until all volumes are created from all the snapshots that are part of a volume group snapshot.
+
+.Prerequisites
+* Logged in to a running {product-title} cluster.
+* PVC has been created using a Container Storage Interface (CSI) driver that supports volume group snapshots.
+* A storage class to provision the storage back end.
+* A volume group snapshot has been created and is ready to use.
+
+.Procedure
+
+To restore existing volumes to a previous state from a volume group snapshot:
+
+. Specify a `VolumeSnapshot` data source from a volume group snapshot for a PVC as shown in the following example:
++
+.Example restore PVC YAML file
+[source, yaml]
+----
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: <pvc-restore-name> <1>
+  namespace: <namespace> <2>
+spec:
+  storageClassName: csi-hostpath-sc
+  dataSource:
+    name: snapshot-62abb5db7204ac6e4c1198629fec533f2a5d9d60ea1a25f594de0bf8866c7947-2023-04-26-2.20.4 <3>
```

Comment:
the name need to be changed to match the new format its still referring to old format of volume snapshot created for a group

Reply:
I think that this is now fixed. PTAL. @Madhu-1